config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

```diff
@@ -355,8 +361,6 @@ postsubmits:
       preset-service-account: "true"
       preset-aws-ssh: "true"
       preset-aws-credential: "true"
-    branches:
-      - release-0.4
```

Comment:
Run against all branches, since https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/1501 will forward port the test to the master branch as well.